src/driver/args.c

```diff
@@ -110,100 +105,116 @@ void yf_parse_args(int argc, char ** argv, struct yf_args * args) {
             continue;
         }
 
-        if (STREQ(arg, "-h") || STREQ(arg, "--help")) {
-            yf_check_action(args, YF_HELP);
-            continue;
-        }
+        if (arg[0] == '-') {
+            ++arg;
+            if (!arg[1]) {
+                if (arg[0] == 'h') {
```

Comment:
But what if the user calls with `yfc -hey-this-is-invalid`?

Reply:
Then `arg[1]` wouldn't be `0` :)